### DISCUSSION TITLE
Add border/outline for ui.menu and ui.popup

### DISCUSSION THREAD

**Discussion by ascarter** [helix-editor/helix#9992] [General] [answered]
An outline border should be used when displaying `ui.menu` and `ui.popup` instead of relying solely on color. This is how the space bar menu and file picker work for example. These work just fine without having a background color to differentiate because the border makes it clear. The theme should support `ui.menu.border` and `ui.popup.border` to set a color like the window border on `ui.window`. I think using `border` makes it clear but whatever term fits is fine. If you want no border, make the color match the background color of `ui.menu`/`ui.popup` and it would disappear.

I have difficulties differentiating color so using styling and things like borders are much easier to perceive than trying to find colors to work. I use a very simple terminal theme that use the least amount of color I can. While I can find something to contrast, it would look much nicer to just have some borders here instead of having to choose a background color.

Example with LSP popup for Rust code. It's pretty much unreadable without using a background color
<img width="1816" alt="Screenshot 2024-03-24 at 11 25 55" src="https://github.com/helix-editor/helix/assets/4931/f791744f-f79a-4f1d-abc1-22e150c2c655">

Example of the popup menu. Note that this is easily distinguishable because of the border and does not require any color background to work.
<img width="1816" alt="Screenshot 2024-03-24 at 11 28 38" src="https://github.com/helix-editor/helix/assets/4931/ef499201-22cb-4d8b-ba46-880bfef14d23">

Example of the file picker also using border. This works easily as well.
<img width="1816" alt="Screenshot 2024-03-24 at 11 28 49" src="https://github.com/helix-editor/helix/assets/4931/66f3f28c-d636-477b-af11-a3138eec5a56">

**Comment by woojiq** [comment, marked as answer]
popup-border
https://docs.helix-editor.com/master/configuration.html

**Comment by ascarter** [comment]
Oh wow - sorry - I either missed that or it is new. Thanks!
